bedrock/firefox/templates/firefox/australis/fx38_0_5/firstrun.html

```diff
@@ -0,0 +1,137 @@
+
+{% block body_class %}mozID{% endblock %}
+
+{% block tabzilla_css %}{% endblock %}
+{% block tabzilla_js %}{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_fx38_0_5_firstrun' %}
+{% endblock %}
+
+{% block site_header %}
+  <header id="masthead">
+    {% block tabzilla_tab %}
+    <div id="tabzilla">
+      <a href="{{ url('mozorg.home') }}">Mozilla</a>
+    </div>
+    {% endblock %}
+    {% block site_header_nav %}{% endblock %}
+    {% block site_header_logo %}{% endblock %}
+    {% block breadcrumbs %}{% endblock %}
+  </header>
+{% endblock %}
+
+{% block content %}
+<section id="intro" data-funnelcake="{{ funnelcake_id }}">
+  <div class="container">
+    <div id="logo">
+      {{ high_res_img('img/firefox/australis/fx38.0.5/firefox-logo.png', {'alt': 'Firefox', 'width': '142', 'height': '147'}) }}
+    </div>
+
+    <div class="leftcol">
+      <h1>
+        {{ _('Almost done!') }}
+      </h1>
+
+      <h2>
+        {# L10n: Line breaks below are for visual formatting only. #}
+        <span class="ctaMenu">{{ _("Set up Sync and <br>you’re good to go.") }}</span>
+        <span class="ctaDirect">{{ _('Sign in to Firefox and <br>you’re good to go.') }}</span>
+      </h2>
+
+      <p>
+        {{ _('With your Firefox Account you can sync your bookmarks, passwords, open tabs and more, and access them everywhere you use Firefox.') }}
+      </p>
+    </div>
+
+    <div class="rightcol">
+      {{ high_res_img('img/firefox/australis/fx38.0.5/sync.png', {'alt': 'Firefox Sync', 'width': '219', 'height': '118'}) }}
+      <button class="button-flat-green" id="cta-signup">
+        <span class="ctaMenu">{{ _('Sign in to get started') }}</span>
+        <span class="ctaDirect">{{ _('Create a Firefox Account') }}</span>
+      </button>
+      <a id="cta-signin" href="https://accounts.firefox.com/signin">
+        <span class="ctaMenu">{{ _('Already have a Firefox Account?') }}</span>
+        <span class="ctaDirect">{{ _('Already have one? Sign in.') }}</span>
+      </a>
+    </div>
+  </div>
+</section>
+
+{% if video_url %}
+<section id="video">
+  <div class="container">
+    <div class="leftcol">
+      <h2 id="video-title">{{ _('From our hands to yours') }}</h2>
+
+      <p>
+        {{ _('Firefox is built from the heart by thousands who believe in personal freedom on the Web.') }}
+      </p>
+
+      <div id="video-container">
+        <div id="video-frame">
+          <video tabindex="0" id="firstrun-video" controls="controls" poster="{{ static('img/firefox/firstrun/video-poster-spring-campaign-2015.jpg') }}" width="100%" height="auto">
+            <source src="{{ video_url }}.webm" type="video/webm">
+            <source src="{{ video_url }}.mp4" type="video/mp4">
+          </video>
+        </div>
+      </div>
+    </div>{#-- /.leftcol --#}
+
+    <div class="rightcol">
+      <a id="cta-video" href="{{ video_url }}.mp4"><span>{{ _('Watch the video') }}</span></a>
```

Comment:
Nit: link format here could be `.webm,` to match the source ordering of the video element above?